data/sql/updates/pending_db_world/rev_1667666190202265500.sql

```diff
@@ -0,0 +1,8 @@
+-- Quest: A Pilgrim's Plight
+UPDATE `gameobject` SET `spawntimesecs`=0 WHERE `guid`=25340 AND `id`=184478;
```

Comment:
Worldserver error: 
Table `gameobject` has gameobject (GUID: 25340 Entry: 184478) with `spawntimesecs` (0) value, but the gameobejct is marked as despawnable at action.

```suggestion
UPDATE `gameobject` SET `spawntimesecs`=1 WHERE `guid`=25340 AND `id`=184478;
```